tests/integration_test.rs

```diff
@@ -109,9 +109,21 @@ async fn test_graph_specs() {
             },
           ));
         }
-        deno_graph::FastCheckTypeModuleSlot::Error(diagnostic) => {
-          output_text
-            .push_str(&indent(&diagnostic.message_with_range_for_test()));
+        deno_graph::FastCheckTypeModuleSlot::Error(diagnostics) => {
+          let message = diagnostics
+            .iter()
+            .map(|d| match d.range() {
+              Some(range) => {
+                format!(
+                  "{}\n    at {}@{}",
+                  d, range.specifier, range.range.start
+                )
+              }
+              None => format!("{}\n    at {}", d, d.specifier()),
+            })
+            .collect::<Vec<_>>()
+            .join("\n");
+          output_text.push_str(&indent(&message));
```

Comment:
How come there's no change in expected output if we're using the new diagnostics system?

Reply:
I kept it the same for now, but will do a follow up PR. This was copied from somewhere else to keep it the same.